app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
+    everyone_has_vac = True
+    persons_without_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            everyone_has_vac = False
```

Comment:
you can simplify your code: when VaccineError happens, just return a message and exit your code. You don't need `everyone_has_vac` variable and an additional check if it is False